### ISSUE TITLE
Add `~/bin` to `PATH` on remote install

### ISSUE THREAD

**Issue by gadamiak** [grantm/bcvi#1]
When installing BCVI on remote host (Ubuntu Server) with `bcvi --install user@hostname` the folder `~/bin`, which holds the executable, was not added to `PATH`. In effect it was not working until I added it manually:

```
## START-BCVI
export PATH="$(echo ~/bin):$PATH"
...
## END-BCVI
```

I guess it should be added by installer or at least be mentioned in installation instructions.

**Comment by gadamiak** [comment]
Actually, I had to move `export...` out of `## ...-BCVI` block as it gets rewritten upon update.

**Comment by grantm** [comment]
Hmm apparently I failed to close this issue when I released the fix in the 3.05 release.
